aspnetcore/blazor/hybrid/routing.md

```diff
@@ -42,13 +42,16 @@ using Microsoft.AspNetCore.Components.WebView;
   
 ## .NET MAUI
 
-Add the event handler to the constructor of the `Page` where the `BlazorWebView` is constructed:
+Add the event handler below to the constructor of the `Page` where the `BlazorWebView` is constructed. This would be `MainPage.xaml.cs` in the default `maui-blazor` template.
```

Comment:
Though it's fairly clear what is meant here, I doubt most users would know the name `maui-blazor`, because it's only ever seen from the CLI. Most users by far use VS.